README.md

```diff
@@ -1,16 +1,32 @@
-# Title
+# Bookend: Git Workflow
 
-## Learning Goals
+## Looking Back
 
--SWBAT 1
--SWBAT 2
+Over the past several lessons, we built on the foundation of our Git basics
```

Comment:
Something here reads a little awkwardly. Maybe "...we continued building on the foundations of Git basics..."